web/package.json

```diff
@@ -8,6 +8,8 @@
     "serve": "vue-cli-service serve --host 0.0.0.0",
     "build": "vue-cli-service build",
     "test": "vue-cli-service test:unit",
+    "check-format": "prettier --check \"{src,tests}/**/!(*@(.html))\"",
+    "format": "prettier --write \"{src,tests}/**/!(*@(.html))\"",
```

Comment:
Would it make more sense to change it to exclude `.imjoy.html`? We don't have regular html files right now, but maybe for future cases?

Reply:
Ok, sounds good!

Reply:
I tested the change on a regular html file, and it works.